doc/UsersGuide/source/fmitlm.rst

```diff
@@ -119,6 +119,9 @@ When importing the model into OMEdit, roughly the following commands will be exe
 .. omc-mos ::
   :erroratend:
 
+
+  loadFile(getInstallationDirectoryPath() + "/share/doc/omc/testmodels/BouncingBall.mo")
+  translateModelFMU(BouncingBall)
```

Comment:
You could make these hidden in an omc-mos section before this if you wanted to conceal it

Reply:
Sure we can do that but I kept them intentionally. A lot users try to mimic the commands from the users guide.